ai/page-index/page-index.js

```diff
@@ -1,12 +1,17 @@
 module.exports = [
   {
-    title: "AI  and Vector: pgai on Tiger",
+    title: "AI  and Vector",
     href: "ai",
     filePath: "index.md",
     pageComponents: ["featured-cards"],
     excerpt:
-      "Information about pgai on TigerData and how to use it.",
+      "Information about pgai on Tiger Data and how to use it.",
     children: [
+      {
+        title: "Integrate Tiger Cloud with your AI Assistant",
+        href: "mcp-server",
+        excerpt: "Manage your services and optimize your schema and queries with your AI Assistant",
+      },
```

Comment:
Does this mean the MCP server will be under the `AI and Vector` category? Is that really the best place for it? Mostly asking because the "except" above makes it sound like this section is mostly about `pgai`. But the MCP server can actually used by anyone to interact with Tiger Cloud, whether or not they're using `pgai`/`pgvector`/`pgvectorscale`.

Reply:
Have changed the excerpt. Will have the EON and tiger workers in this section as well.